lib/Conversion/StableHLOToTTIR/StableHLOToTTIR.cpp

```diff
@@ -19,9 +19,8 @@
 #include <mlir/Pass/Pass.h>
 #include <mlir/Support/LogicalResult.h>
 #include <mlir/Transforms/DialectConversion.h>
-
+#ifdef TTMLIR_ENABLE_STABLEHLO
```

Comment:
Do we need guards in the cpp files?  Ideally this file is never compiled / referenced by cmake if stablehlo is disabled.